packages/azure-cosmos-db/azure-cosmos-db.0.2.0/opam

```diff
@@ -0,0 +1,38 @@
+opam-version: "2.0"
+license: "MIT"
+synopsis: "Azure cosmos db interface"
+description: """
+Provides an interface to the Azure Cosmos db.
+"""
+maintainer: "Morten Knaack <knaack@gmail>"
+authors: "Morten Knaack <knaack@gmail>"
+depends: [
+  "ocaml" {>= "4.08.0"}
+  "dune" {>= "2.7.0"}
+  "lwt"
+  "lwt_ppx"
+  "lwt_ssl"
+  "ssl"
+  "atdgen"
+  "base64" {>= "3.5.0"}
+  "uri"
+  "cohttp"
+  "cohttp-lwt-unix"
+  "cryptokit"
+  "alcotest-lwt" {with-test}
+]
+homepage: "https://bitbucket.org/knaack/azure-cosmos-db"
```

Comment:
```suggestion
conflicts: [ "result" {< "1.5"} ]
homepage: "https://bitbucket.org/knaack/azure-cosmos-db"
```

Reply:
Due to the lower bound failure
```
#       ocamlc src/.cosmos.objs/byte/cosmos__Databases.{cmo,cmt} (exit 2)
# (cd _build/default && /home/opam/.opam/4.08/bin/ocamlc.opt -w -40 -g -bin-annot -I src/.cosmos.objs/byte -I /home/opam/.opam/4.08/lib/astring -I /home/opam/.opam/4.08/lib/atd -I /home/opam/.opam/4.08/lib/atdgen -I /home/opam/.opam/4.08/lib/base -I /home/opam/.opam/4.08/lib/base/caml -I /home/opam/.opam/4.08/lib/base/shadow_stdlib -I /home/opam/.opam/4.08/lib/base64 -I /home/opam/.opam/4.08/lib/biniou -I /home/opam/.opam/4.08/lib/bytes -I /home/opam/.opam/4.08/lib/cohttp -I /home/opam/.opam/4.08/lib/cohttp-lwt -I /home/opam/.opam/4.08/lib/cohttp-lwt-unix -I /home/opam/.opam/4.08/lib/conduit -I /home/opam/.opam/4.08/lib/conduit-lwt -I /home/opam/.opam/4.08/lib/conduit-lwt-unix -I /home/opam/.opam/4.08/lib/cryptokit -I /home/opam/.opam/4.08/lib/easy-format -I /home/opam/.opam/4.08/lib/fieldslib -I /home/opam/.opam/4.08/lib/fmt -I /home/opam/.opam/4.08/lib/ipaddr -I /home/opam/.opam/4.08/lib/ipaddr/unix -I /home/opam/.opam/4.08/lib/logs -I /home/opam/.opam/4.08/lib/lwt -I /home/opam/.opam/4.08/lib/lwt/unix -I /home/opam/.opam/4.08/lib/lwt_ssl -I /home/opam/.opam/4.08/lib/magic-mime -I /home/opam/.opam/4.08/lib/mmap -I /home/opam/.opam/4.08/lib/ocaml/threads -I /home/opam/.opam/4.08/lib/parsexp -I /home/opam/.opam/4.08/lib/ppx_sexp_conv/runtime-lib -I /home/opam/.opam/4.08/lib/re -I /home/opam/.opam/4.08/lib/re/posix -I /home/opam/.opam/4.08/lib/result -I /home/opam/.opam/4.08/lib/seq -I /home/opam/.opam/4.08/lib/sexplib -I /home/opam/.opam/4.08/lib/sexplib0 -I /home/opam/.opam/4.08/lib/ssl -I /home/opam/.opam/4.08/lib/stringext -I /home/opam/.opam/4.08/lib/uchar -I /home/opam/.opam/4.08/lib/uri -I /home/opam/.opam/4.08/lib/uri/services -I /home/opam/.opam/4.08/lib/uri/sexp -I /home/opam/.opam/4.08/lib/yojson -I /home/opam/.opam/4.08/lib/zarith -intf-suffix .ml -no-alias-deps -open Cosmos -o src/.cosmos.objs/byte/cosmos__Databases.cmo -c -impl src/databases.pp.ml)
# File "src/databases.ml", line 176, characters 33-42:
# 176 |     if code = expected_code then Result.ok expected_code
#                                        ^^^^^^^^^
# Error: Unbound value Result.ok
#     ocamlopt src/.cosmos.objs/native/cosmos__Databases.{cmx,o} (exit 2)
# (cd _build/default && /home/opam/.opam/4.08/bin/ocamlopt.opt -w -40 -g -I src/.cosmos.objs/byte -I src/.cosmos.objs/native -I /home/opam/.opam/4.08/lib/astring -I /home/opam/.opam/4.08/lib/atd -I /home/opam/.opam/4.08/lib/atdgen -I /home/opam/.opam/4.08/lib/base -I /home/opam/.opam/4.08/lib/base/caml -I /home/opam/.opam/4.08/lib/base/shadow_stdlib -I /home/opam/.opam/4.08/lib/base64 -I /home/opam/.opam/4.08/lib/biniou -I /home/opam/.opam/4.08/lib/bytes -I /home/opam/.opam/4.08/lib/cohttp -I /home/opam/.opam/4.08/lib/cohttp-lwt -I /home/opam/.opam/4.08/lib/cohttp-lwt-unix -I /home/opam/.opam/4.08/lib/conduit -I /home/opam/.opam/4.08/lib/conduit-lwt -I /home/opam/.opam/4.08/lib/conduit-lwt-unix -I /home/opam/.opam/4.08/lib/cryptokit -I /home/opam/.opam/4.08/lib/easy-format -I /home/opam/.opam/4.08/lib/fieldslib -I /home/opam/.opam/4.08/lib/fmt -I /home/opam/.opam/4.08/lib/ipaddr -I /home/opam/.opam/4.08/lib/ipaddr/unix -I /home/opam/.opam/4.08/lib/logs -I /home/opam/.opam/4.08/lib/lwt -I /home/opam/.opam/4.08/lib/lwt/unix -I /home/opam/.opam/4.08/lib/lwt_ssl -I /home/opam/.opam/4.08/lib/magic-mime -I /home/opam/.opam/4.08/lib/mmap -I /home/opam/.opam/4.08/lib/ocaml/threads -I /home/opam/.opam/4.08/lib/parsexp -I /home/opam/.opam/4.08/lib/ppx_sexp_conv/runtime-lib -I /home/opam/.opam/4.08/lib/re -I /home/opam/.opam/4.08/lib/re/posix -I /home/opam/.opam/4.08/lib/result -I /home/opam/.opam/4.08/lib/seq -I /home/opam/.opam/4.08/lib/sexplib -I /home/opam/.opam/4.08/lib/sexplib0 -I /home/opam/.opam/4.08/lib/ssl -I /home/opam/.opam/4.08/lib/stringext -I /home/opam/.opam/4.08/lib/uchar -I /home/opam/.opam/4.08/lib/uri -I /home/opam/.opam/4.08/lib/uri/services -I /home/opam/.opam/4.08/lib/uri/sexp -I /home/opam/.opam/4.08/lib/yojson -I /home/opam/.opam/4.08/lib/zarith -intf-suffix .ml -no-alias-deps -open Cosmos -o src/.cosmos.objs/native/cosmos__Databases.cmx -c -impl src/databases.pp.ml)
# File "src/databases.ml", line 176, characters 33-42:
# 176 |     if code = expected_code then Result.ok expected_code
#                                        ^^^^^^^^^
# Error: Unbound value Result.ok
```

Reply:
Thanks for looking into this. I have updated the PR.